src/user/user.service.ts

```diff
@@ -29,13 +29,17 @@ export class UserService {
   }
 
   async findEmailLogin(email: string) {
-    try {
-      return await this.repo.findOneOrFail({
-        where: { email }
-      });
-    } catch (err) {
-      throw new Error(err);
-    }
+    return await this.repo.findOneOrFail({
+      where: { email }
+    });
+  }
+
+  async findUserByIdentityCard(identity_card_number: string) {
```

Comment:
```suggestion
  async findUserByIdentityCard(identityCardNumber: string) {
```

Chú ý ngoại trừ db field thì các biến viết theo camelCase